language/diem-framework/transactional-tests/tests/prologue/fail_if_cant_pay_gas.move

```diff
@@ -0,0 +1,39 @@
+//# init --addresses Alice=0x76c3623ac80c38e2f1f4e90f07c8e21a
+//#      --private-keys Alice=1db3dd81f19eb4a22028036e05043a3b42415e2297439195c804744fd16ecd6a
+
+
+//# run --signers 0xA550C18
+//#     --private-key DiemRoot
+script {
+    use DiemFramework::DiemAccount::create_validator_account;
+
+    fun main(s: signer) {
+        create_validator_account(&s, @Alice, x"35830210d93ec6a22db9394a7367f72d", x"40");
+    }
+}
+
+
+//# run --signers 0x76c3623ac80c38e2f1f4e90f07c8e21a
+//#     --private-key Alice
```

Comment:
@tnowacki I've implemented the private key mapping. I feel like it would be better if could also use named addresses for signers. Thoughts?